### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by xcypcpf** [argusswift/YOLOv4-pytorch#35]
hello:
        what is the final training loss when you train voc dataset. 
        thanks

**Comment by argusswift** [comment]
> hello:
> what is the final training loss when you train voc dataset.
> thanks

6-7
